cfgov/v1/util/migrations.py

```diff
@@ -57,3 +61,73 @@ def get_or_create_page(apps, page_cls_app, page_cls_name, title, slug,
     parent_page.save()
 
     return page
+
+
+def get_stream_data(page_or_revision, field_name):
+    """ Get the stream field data for a given field name on a page or a
+    revision """
+    try:
+        # If page_or_revision is a page, this will work
+        field = getattr(page_or_revision, field_name)
+        stream_data = field.stream_data
+    except AttributeError:
+        # Otherwise it will raise an Attribute error and we can assume
+        # page_or_revision is a revision
+        revision_content = json.loads(page_or_revision.content_json)
+        field = revision_content[field_name]
+        stream_data = json.loads(field)
+    return stream_data
```

Comment:
I think I prefer sticking with the `try`/`except`... it seems more Pythonic, unless you (or anyone else) is strongly in favor of using `isinstance`.

That said, this code wouldn't really be very pretty either way.